src/api/java/com/minecolonies/api/colony/requestsystem/request/IRequest.java

```diff
@@ -216,22 +219,16 @@ default AssigningStrategy getStrategy()
     boolean canBeDelivered();
 
     /**
-     * Method to get the ItemStack used for the getDelivery.
+     * Method to get the ItemStacks used for the delivery.
      *
-     * @return The ItemStack that the Deliveryman transports around. ItemStack.Empty means no delivery possible.
+     * @return The ItemStacks that the Deliveryman transports around. {@link NonNullList#isEmpty()} means no delivery possible.
      */
     @NotNull
-    ItemStack getDelivery();
+    ImmutableList<ItemStack> getDeliveries();
 
-    /**
-     * Method used to set the delivery on a Request.
-     * Should set the result.
-     * <p>
-     * Noop this if it is not supported.
-     *
-     * @param delivery The delivery that is being made.
-     */
-    void setDelivery(@Nullable final ItemStack delivery);
+    void overrideCurrentDeliveries(@NotNull final ImmutableList<ItemStack> stacks);
```

Comment:
Why did this javadoc get deleted?
And why does the other method have no javadoc?